src/pages/Login/Input/Input.js

```diff
@@ -0,0 +1,24 @@
+/* eslint-disable */
+
+import React from 'react';
+import '../Input/Input.scss';
+
+const Input = props => {
+  return (
+    <>
+      <input
+        className={
+          props.isAlertPop ? 'loginInput' && 'alertInput' : 'loginInput'
```

Comment:
```suggestion
          props.isAlertPop ? 'alertInput' : 'loginInput'
```
값이 있는 string은 truthy값이기에 무조건 && 구문을 통과하게 되네요!

Reply:
그러네요 ㅋㅋㅋㅋㅋ